lib/codit/outbreak.py

```diff
@@ -59,7 +59,12 @@ def simulate(self):
             self.record_state()
         self.recorder.realized_r0 = self.pop.realized_r0()
         self.recorder.society_config = self.society.cfg
-        self.recorder.disease_config = self.disease.cfg
+
+        if type(self.disease) is set:
```

Comment:
should this not be called `self.diseases` now? And always be a set, potentially a singleton?